tests/test_swtpm_setup_overwrite

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+# For the license, see the LICENSE file in the root directory.
+
+ROOT=${abs_top_builddir:-$(dirname "$0")/..}
+TESTDIR=${abs_top_testdir:-$(dirname "$0")}
+SRCDIR=${abs_top_srcdir:-$(dirname "$0")/..}
+
+source ${TESTDIR}/common
+skip_test_no_tpm12 "${SWTPM_EXE}"
+statebasename="tpm-00.permall"
+
+SWTPM_SETUP_CONF=$SRCDIR/samples/swtpm_setup.conf
+
+trap "cleanup" SIGTERM EXIT
+
+function cleanup()
+{
+	rm -rf ${workdir}
```

Comment:
Please use tab for indentation like in the other test cases..

Reply:
Sure, I'll also fix test_print_states as well.